handler/transport_test.go

```diff
@@ -56,6 +56,7 @@ func Test_transport_RoundTrip(t *testing.T) {
 		cfg                  config.Proxy
 		noAuthPaths          []*policy.Assertion
 		insecureCipherSuites []*tls.CipherSuite
+		metrics              service.Metrics
```

Comment:
Please add called and skipped test case for `t.metrics.Observe()`. If accessing the field of the actual object in test is difficult, please create a mock of `service.Metrics` interface.

Reply:
@WindzCUHK add test case in https://github.com/AthenZ/authorization-proxy/pull/30/commits/724c71a7ccdf857e4075e21b21d812c2868631f9